web/src/main/java/org/akaza/openclinica/control/submit/OCRequestWrapper.java

```diff
@@ -0,0 +1,80 @@
+package org.akaza.openclinica.control.submit;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletRequestWrapper;
+
+/***
+ * Request wrapper enabling the update of a request-parameter. 
+ *
+ */
+final class OCRequestWrapper
```

Comment:
why do you need to update request parameters?

Reply:
because I need keep the previous table configurations and pass to next UI page, and for the 3rd party lib:
new TableFacadeImpl(getTableName(), request);
It was passed in by request parameters

Reply:
It looks like you can set the attributes on TableFacadeImpl directly. You could fetch the settings from the session and set them directly on that object in ListStudySubjectTableFactory.java. Look at SDVUtil for example

Reply:
Here I have different implementation, new PR:
https://github.com/OpenClinica/OpenClinica/pull/2885, if go with new PR, we can close this PR